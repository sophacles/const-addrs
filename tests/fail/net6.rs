use const_addrs::net6;

fn main() {
    let _ = net6!("2001:db8::32::23/129");
    let _ = net6!("2001:db8::32:23/129");
}
