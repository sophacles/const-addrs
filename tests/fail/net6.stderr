error: invalid address: invalid IPv6 address syntax
 --> tests/fail/net6.rs:4:19
  |
4 |     let _ = net6!("2001:db8::32::23/129");
  |                   ^^^^^^^^^^^^^^^^^^^^^^

error: invalid prefix
 --> tests/fail/net6.rs:5:19
  |
5 |     let _ = net6!("2001:db8::32:23/129");
  |                   ^^^^^^^^^^^^^^^^^^^^^
